common/components/GasSlider/components/SimpleGas.scss

```diff
@@ -34,3 +50,26 @@
     }
   }
 }
+
+.fade {
```

Comment:
We should add a slight transition delay on entry here (~300ms.) I noticed that this loader would pop up _every change I made_, even if there was no gas request made to the node. Either that, or we should make it so it only goes into PENDING state on network request.

Reply:
I don't think delaying it will help, it just pushed back the problem. The conditions for the spinner were wrong when I commited, this has been fixed now so it shouldn't show up on every change